mungegithub/github/github.go

```diff
@@ -1786,6 +1786,26 @@ func (obj *MungeObject) AddAssignee(owner string) error {
 	return nil
 }
 
+// IsAssignee will determine if the owner is a valid assignee in this repo
+func (obj *MungeObject) IsAssignee(owner string) (bool, error) {
```

Comment:
MungeObjects are Issues or PRs and this method is unrelated to either of those so I can't see a reason to add a dependency on an arbitrary MungeObject.